server/src/main.rs

```diff
@@ -36,7 +36,11 @@ use handlers::*;
 
 #[tokio::main]
 async fn main() {
-    dotenv().expect("Failed to read .env");
```

Comment:
panicing here means a .env file *must be present* even though passing env vars directly should be a perfectly valid way to start the server